src/diffusers/pipelines/pipeline_utils.py

```diff
@@ -1560,6 +1570,8 @@ def progress_bar(self, iterable=None, total=None):
                 f"`self._progress_bar_config` should be of type `dict`, but is {type(self._progress_bar_config)}."
             )
 
+        if self._progress_bar_config.get('disable', False) == True:
+            return NotTQDMNoOp()
         if iterable is not None:
             return tqdm(iterable, **self._progress_bar_config)
```

Comment:
instead, you can do this:

```py
@torch._dynamo.disable
def progress_bar(iterable, **kwargs):
    return tqdm(iterable, **kwargs)
```
and then instead of returning tqdm, you:
```py
return progress_bar(...)
```

Reply:
(or `@torch._dynamo.skip` if it provides better behaviour)

Reply:
Oooh I see `torch.compiler.disable` based on https://pytorch.org/docs/stable/torch.compiler_fine_grain_apis.html

Reply:
This PR is a one-line add thank you :)